packages/rum-core/src/performance-monitoring/performance-monitoring.js

```diff
@@ -102,24 +102,29 @@ class PerformanceMonitoring {
       ) {
         let target = task.target
         let name = target.getAttribute('name')
-        let classes = target.getAttribute('class')
 
         let additionalInfo = ''
         if (name) {
           additionalInfo = `["${name}"]`
-        } else if (classes) {
-          classes = classes.split(' ').join('.')
-          additionalInfo = `.${classes}`
         }
+
         let tagName = target.tagName.toLowerCase()
-        transactionService.startTransaction(
+        let tr = transactionService.startTransaction(
           `Click >> ${tagName}${additionalInfo}`,
           USER_INTERACTION,
           {
             managed: true,
-            canReuse: true
+            canReuse: true,
+            reuseThreshold: 100
           }
         )
+
+        if (tr) {
+          let classes = target.getAttribute('class')
```

Comment:
Thought about this once again, Its good idea to truncate this or not add this if the length of the classnames exceed a certain threshold. WDYT about like this
```
target.classList.value.length > 250 // dont add the context
```

This is a guess that, user is relying on auto generation and wont be of any help. Atleast we are reducing the payload size

Reply:
Let's added a generic truncation for all custom contexts separately.

Reply:
Custom contexts doesn't require truncation as far as I remember, This is a edge case and we can handle it now in this PR or atleast in follow up PR as it creates noise since the feature is enabled by default. Can you add a task to the original isssue.

Reply:
To be honest, I'm not sure what exactly we're trying to solve here, even the autogenerated classes in most cases are far less than 250, so to avoid them we would have to set the limit much lower than this. 

Re. generic custom context limit, we already have an issue for it: #46